Watchman.Discord/Framework/Server.cs

```diff
@@ -8,16 +9,18 @@ namespace Watchman.Discord.Framework
     public static class Server
     {
         private static DiscordSocketClient _client;
+        private static IMongoDatabase _database;
 
         public static bool Initialized = false;
```

Comment:
Dlaczego public, a nie private?

Reply:
Ponieważ Initialized to informacja o obiekcie, która spokojnie może być widoczna z zewnątrz
Na razie nie jest używana na zewnątrz, ale nic nie stoi na przeszkodzie żeby to pokazywać